posix.mak

```diff
@@ -126,6 +126,15 @@
 #  in the build folder `.generated`, s.t. Ddoc can be run on the modified sources.
 #
 #  See also: https://dlang.org/blog/2017/03/08/editable-and-runnable-doc-examples-on-dlang-org
+#
+#  Custom DDoc wrapper
+#  -------------------
+#
+#  `ddoc.d` is a wrapper around Ddoc and allows expanding Ddoc macros dynamically
+#  before actually running Ddoc.
+#  Currently this is used for:
+#    - TOC
+#    - assert -> writeln magic
```

Comment:
Seems like the two lists of the program's tasks/capabilities are redundant (and one is more complete than the other).

:+1: on more docs though.